apps/model/model.js

```diff
@@ -806,17 +788,18 @@ async function showInfo() {
             td = row.insertCell();
             td.innerHTML = date;
             td = row.insertCell();
-            td.innerHTML = '<button class="btn btn-primary btn-xs my-xs-btn" '+
-            'id="removeModel" type="button"><i class="material-icons"'+
+            td.innerHTML = '<button class="btn-del" '+
+            'id=removeModel'+ modelCount+' type="button"><i class="material-icons"'+
             'style="font-size:16px;">delete_forever</i>Remove Model</button>';
             td = row.insertCell();
-            td.innerHTML = '<button class="btn btn-primary btn-xs my-xs-btn" '+
-            'id="chngClassListBtn" type="button"><i class="material-icons"' +
+            td.innerHTML = '<button class="btn-change" '+
+            'id=chngClassListBtn'+ modelCount+' type="button"><i class="material-icons"' +
             'style="font-size:16px;">edit</i>  Edit Classes</button>';
-            document.getElementById('removeModel').addEventListener('click', () => {
+            document.getElementById('removeModel'+modelCount).addEventListener('click', () => {
               deleteModel(name);
             });
-            document.getElementById('chngClassListBtn').addEventListener('click', () => {
+            modelCount+=1;
+            document.getElementById('chngClassListBtn'+modelCount).addEventListener('click', () => {
               showNewClassInput(name);
             });
```

Comment:
A little issue, but I think swapping these lines will resolve it.
```suggestion
            document.getElementById('chngClassListBtn'+modelCount).addEventListener('click', () => {
              showNewClassInput(name);
            });
            modelCount+=1;
```